lib/gst/player/gstplayer.c

```diff
@@ -1443,10 +1450,18 @@ gst_player_seek_internal_locked (GstPlayer * self)
 
   remove_tick_source (self);
   self->priv->is_eos = FALSE;
+  
+  start_pos = gst_player_get_position(self);
```

Comment:
This now breaks seeks, you would now always seek to the current position. The seek function needs some refactoring to handle both cases.